### PR TITLE
Add 'kpod images' and 'kpod rmi' commands

### PR DESCRIPTION
**Pull request by 14rcole**
'kpod images' lists all images on a system.  'kpod rmi' removes one or more images from a system.  The images will not be removed if they are associated with a running container, unless the -f option is used

### CODE REVIEW

File: completions/bash/kpod [hunk 1]
```diff
@@ -2,6 +2,34 @@
...
+_kpod_images() {
+    local boolean_options="
+     --help
+     -h
+     --quiet
+     -q
+     --noheading
+     -n
+     --no-trunc
+     --digests
```
Comment (rhatdan) [inline completions/bash/kpod]: --format?

Comment (rhatdan) [inline completions/bash/kpod]: --filter
-f

File: docs/kpod-images.1.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+**--quiet, -q**
+
+Lists only the image IDs.
+
+**--digests**
+
+Show image digests
+
+## EXAMPLE
```
Comment (rhatdan) [inline docs/kpod-images.1.md]: Missing
--format 
--filter, -f

File: docs/kpod-rmi.1.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+**kpod** **rmi** **imageID [...]**
+
+## DESCRIPTION
+Removes one or more locally stored images.
+
+## OPTIONS
+
+**--force, -f**
+
+Forcibly remove the image
```
Comment (rhatdan) [inline docs/kpod-rmi.1.md]: Executing this command will stop all containers that are using the image and remove them from the system?

File: docs/kpod.1.md [hunk 4]
```diff
@@ -32,6 +32,12 @@ has the capability to debug pods/images created by crio.
 ## launch
 Launch a pod
 
+## images
```
Comment (rhatdan) [inline docs/kpod.1.md]: Alphabatize

File: cmd/kpod/main.go [hunk 5]
```diff
@@ -14,7 +14,29 @@ func main() {
...
 	app.Commands = []cli.Command{
+		imagesCommand,
 		launchCommand,
+		rmiCommand,
+	}
+	app.Flags = []cli.Flag{
+		cli.StringFlag{
+			Name:  "root",
+			Usage: "path to the root directory in which data, including images,  is stored",
+			Value: "/var/lib/containers/storage",
```
Comment (rhatdan) [inline cmd/kpod/main.go]: These probably should not be constants, I think you can get these values from container storage.

File: cmd/kpod/main.go [hunk 6]
```diff
@@ -14,7 +14,29 @@ func main() {
...
+		},
+		cli.StringFlag{
+			Name:  "runroot",
+			Usage: "path to the 'run directory' where all state information is stored",
+			Value: "/var/run/containers/storage",
```
Comment (rhatdan) [inline cmd/kpod/main.go]: Ditto.

File: cmd/kpod/main.go [hunk 7]
```diff
@@ -14,7 +14,29 @@ func main() {
...
+		},
+		cli.StringFlag{
+			Name:  "storage-driver, s",
```
Comment (rhatdan) [inline cmd/kpod/main.go]: Ditto

Tell us the default.

**Comment by rhatdan** [comment]
Need some tests to prove that these work and make sure nothing breaks them.

**Comment by rhatdan** [comment]
@nalind PTAL

**Comment by rhatdan** [comment]
:+1:

**Comment by rhatdan** [comment]
@mrunalp @runcom PTAL.  Would like to get this in, so that atomic can start to use them.

**Comment by 14rcole** [comment]
Should merging this PR wait until I can add the fix for [projectatomic/buildah#133](https://github.com/projectatomic/buildah/issues/133)?  It's almost finished

**Comment by mrunalp** [comment]
Is this ready now?

**Comment by rhatdan** [comment]
LGTM, I think this is ready to merge.

**Comment by mrunalp** [comment]
retest

**Comment by 14rcole** [comment]
@mrunalp is that a command for a bot or is there something I'm supposed to do?

**Comment by 14rcole** [comment]
retest

**Comment by 14rcole** [comment]
@nalind @mrunalp @runcom If there are no other issues, can this be merged or are you waiting until the rh-atomic-bot tests are working again?

**Comment by mrunalp** [comment]
Needs rebase.

**Comment by rhatdan** [comment]
@14rcole Need to rebase this pull request.

**Comment by rhatdan** [comment]
@mrunalp I think this is ready to merge as soon as tests pass.

**Comment by mrunalp** [comment]
retest

**Comment by mrunalp** [comment]
Needs rebase as I merged another PR.

**Comment by 14rcole** [comment]
@mrunalp done

**Comment by mrunalp** [comment]
retest

**Comment by mrunalp** [comment]
LGTM

**Pull request merged**